plugins/com.google.cloud.tools.eclipse.appengine.facets.test/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineFlexFacetTest.java

```diff
@@ -27,33 +27,36 @@
 import org.mockito.Mock;
 import org.mockito.runners.MockitoJUnitRunner;
 
+// Flex facet temporarily disabled
 @RunWith(MockitoJUnitRunner.class)
 public class AppEngineFlexFacetTest {
   @Mock private IFacetedProject facetedProject;
 
   @Test
+  public void testFlexFacetDoesNotExists() {
+    Assert.assertFalse(
+        ProjectFacetsManager.isProjectFacetDefined("com.google.cloud.tools.eclipse.appengine.facets.flex"));
+  }
+
   public void testFlexFacetExists() {
     Assert.assertTrue(
         ProjectFacetsManager.isProjectFacetDefined("com.google.cloud.tools.eclipse.appengine.facets.flex"));
   }
 
-  @Test
   public void testHasAppEngineFacet_withFacet() {
```

Comment:
Honestly, disabling tests makes me queasy. What might be called for here is a branch with the flex development. The head branch shouldn't have these tests.